app/models/document.rb

```diff
@@ -370,6 +370,10 @@ def custom_error_message_fields
     []
   end
 
+  def self.format_specific_fields
```

Comment:
Is it possible to delete `FORMAT_SPECIFIC_FIELDS = format_specific_fields` from the models altogether, and have the `Document.initialize` method call this internally?

Reply:
This is the exact issue I came across when trying to move everything into Document.rb 😢